apps/tables/package.json

```diff
@@ -1,24 +1,22 @@
 {
   "name": "tables",
-  "version": "0.1.0",
+  "version": "0.0.0",
```

Comment:
Version downgrade?

Reply:
ditto. Version numbering is from elsewhere